app/javascript/packs/application.js

```diff
@@ -14,6 +14,7 @@ require("../includes/vote.js")
 require("../includes/backtotop.js")
 require("../includes/infinitescroll.js")
 require("../includes/navbar.js")
+require("../includes/closediscussion.js")
```

Comment:
File name does not follow our conventions. Two words cannot be used like this, remember?

Reply:
Ok Magesh